projects/packages/videopress/src/client/components/icons/crossed-eye-icon.js

```diff
@@ -1,7 +1,7 @@
 import { SVG, Path } from '@wordpress/components';
 
 const crossedEye = (
-	<SVG width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I am curious about this change, is it something that you detected or these are generated files?

Reply:
I think those were copied from Figma, or maybe sent by the designers and adapted.
I changed the privacy settings to `true` manually and even then it did not show, so I had to inspect the page and change the icon. One of the icons, `play-icon` has a `fill="white"` in it, which made me check the other icons and only then I noticed the `fill="none"`. Removing it solved the problem.

If you take a look at one of the icons on `@wordpress/icons`, like `chevronDown`, you'll see there is no hardcoded `fill` there.